packages/components/Atoms/Icon/TextIcon.vue

```diff
@@ -5,15 +5,22 @@
     <span v-else class="rpl-text-icon__group"><rpl-icon v-bind="iconProps" />{{ text }}</span>
   </span>
   <span v-else-if="text && symbol && placement === 'after'">
-    <span v-if="textWordCount > 1">{{ textWithoutLastWord }}</span>
-    <span v-if="textWordCount > 1" class="rpl-text-icon__group"> {{ textLastWord }}<rpl-icon v-bind="iconProps" /></span>
+    <span v-if="textWordCount > 1">
+      <span>{{ textWithoutLastWord }}</span>
+      <span class="rpl-text-icon__group"> {{ textLastWord }}<rpl-icon v-bind="iconProps" /></span>
+    </span>
+    <span v-else-if="textWordCount === 1 && textIsAbsoluteUrl" class="rpl-text-icon__url-icon">
+      <div class="rpl-text-icon__absolute-url">{{ text }}</div>
```

Comment:
Hmm this is tricky as text props can accept a group of words or just URL or both. Can you try this css setting 
```
overflow-wrap: break-word;
word-wrap: break-word;
```
It should break long words and respect the parent's width.
I found sample code in here https://codepen.io/team/css-tricks/pen/RWaNxr

Reply:
Also can we update unit test with this test case please `text-icon.test.js`